core/src/main/scala/kafka/admin/AclCommand.scala

```diff
@@ -91,10 +93,12 @@ object AclCommand {
     }
   }
 
-  private def removeAcl(opts: AclCommandOptions) {
+  def removeAcl(opts: AclCommandOptions) {
```

Comment:
For testing. Can't rely on `AclCommand.main` as it calls System.exit on getting an exception. `testInvalidPrincipalTypeWhileRemovingAcls` should probably use `removeAcl`.